sycl/test-e2e/Basic/submit_time.cpp

```diff
@@ -1,34 +1,34 @@
 // RUN: %{build} -o %t.out
-// RUN: %{run} %t.out
+// There is an issue with reported device time for the L0 backend, works only on
+// pvc for now. No such problems for other backends.
+// RUN: %if (!ext_oneapi_level_zero || gpu-intel-pvc) %{ %{run} %t.out %}
 
-// Check that device_impl::isGetDeviceAndHostTimerSupported() is not spoiling
-// device_impl::MDeviceHostBaseTime values used for submit timestamp
-// calculation.
+// Check that submission time is calculated properly.
 
 #include <sycl/sycl.hpp>
 
 using namespace sycl;
 
 int main(void) {
-  sycl::queue queue(
-      sycl::property_list{sycl::property::queue::enable_profiling()});
-  sycl::event event = queue.submit([&](sycl::handler &cgh) {
-    cgh.parallel_for<class set_value>(sycl::range<1>{1024},
-                                      [=](sycl::id<1> idx) {});
-  });
+  queue q({property::queue::enable_profiling{}});
+  int *data = malloc_host<int>(1024, q);
 
-  // SYCL RT internally calls device_impl::isGetDeviceAndHostTimerSupported()
-  // to decide how to calculate "submit" timestamp - either using backend API
-  // call or using fallback implementation.
-  auto submit =
-      event.get_profiling_info<sycl::info::event_profiling::command_submit>();
-  auto start =
-      event.get_profiling_info<sycl::info::event_profiling::command_start>();
-  auto end =
-      event.get_profiling_info<sycl::info::event_profiling::command_end>();
```

Comment:
Nit; We try to discourage the use of `using namespace sycl;` so maybe we should remove that instead of removing the qualifications... Just to be good role models. 😉